spec/spec_helper.rb

```diff
@@ -101,31 +103,132 @@
   Kernel.srand config.seed
 end
 
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Layout/SpaceInsideArrayLiteralBrackets
+# rubocop:disable Layout/ExtraSpacing
```

Comment:
move these around the multi-level array section only?

Reply:
moved